tests/Statable/StatableArticle.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Sebdesign\SM\Test\Statable;
+
+use Sebdesign\SM\Traits\Statable;
+use Illuminate\Database\Eloquent\Model;
+
+class StatableArticle extends Model
+{
+    use Statable;
+
+    protected $table = 'articles';
+
+    protected $guarded = [];
+
+    protected $SMConfig = 'graphA'; // the SM graph to use
```

Comment:
The graph is always resolved from the configuration, it should not be hardcoded in the model.

Reply:
This should be resolved with a `getGraph` method, not a property.